reflect_util.go

```diff
@@ -45,12 +46,14 @@ func convertToSlice(v interface{}) ([]interface{}, error) {
 	return s, nil
 }
 
-// Field is a struct field
+// Field is a struct field that represents a single entity of an object.
```

Comment:
Multiline comment

Reply:
Same as earlier. Line syntax is the standard idiomatic way to go.